src/Symfony/Component/Debug/FatalErrorHandler/UndefinedMethodFatalErrorHandler.php

```diff
@@ -36,8 +36,12 @@ public function handleError(array $error, FatalErrorException $exception)
 
         $message = sprintf('Attempted to call an undefined method named "%s" of class "%s".', $methodName, $className);
 
+        if (null === $methods = get_class_methods($className)) {
```

Comment:
I didn't check the implementation of `get_class_methods`, however the API docs. leave it open on when an error can happen. I've update the code in a way we've the best of the two solution, i.e. we do the `class_exists` check first but still check the result of `get_class_methods`. This way the code will keep working even if `get_class_methods` might return `null` later on because of changes in the implementation. Let me know what you think!